cmd/crowdsec-cli/watchers.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/crowdsecurity/crowdsec/pkg/database"
+	"github.com/enescakir/emoji"
+	"github.com/go-openapi/strfmt"
+	"github.com/olekukonko/tablewriter"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var machineID string
+var machinePassword string
+var machineIP string
+var interactive bool
+
+func NewWatchersCmd() *cobra.Command {
+	/* ---- DECISIONS COMMAND */
+	var cmdWatchers = &cobra.Command{
+		Use:   "watchers [action]",
+		Short: "Manage local API watchers",
+		Long: `
```

Comment:
maybe in the "long" description we should clearly differentiate that the API keys are read-only while those can write ? as we're re-using the term "API" I'm afraid we will confuse users.